stix_shifter_modules/cybereason/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,804 @@
+      "value": [
+        "User.emailAddress",
+        "User.adMail"
+      ]
+    }
+  },
+  "autonomous-system": {
+    "fields": {
+      "name": [
+        "Machine.adCompany",
+        "Machine.adOrganization",
+        "Machine.adDepartment",
+        "Machine.adOU"
+      ],
+      "number": [
+        "User.numberOfMachines"
+      ]
+    }
+  },
+  "domain-name": {
+    "fields": {
+      "value": [
+        "Connection.domainName",
+        "Connection.urlDomains"
+      ]
+    }
+  },
+  "url": {
+    "fields": {
+      "value": [
+        "Proxy.pacUrl"
+      ]
+    }
+  },
+   "mac-addr": {
+    "fields": {
+      "value": [
+        "NetworkInterface.macAddressFormat"
+      ]
+    }
+  },
+  "process": {
+    "fields": {
+      "command_line": [
+        "Process.commandLine",
+        "Process.decodedCommandLine"
+      ],
+      "created": [
+        "Process.creationTime",
+        "DetectionEvents.firstSeen"
+      ],
+      "pid": [
+        "Process.applicablePid"
+      ],
+      "name": [
+	    "Process.elementDisplayName",
+        "Process.parentProcess"
+      ],
+      "creator_user_ref.user_id": [
+        "Process.user"
+      ],
+      "parent_ref.pid": [
+        "Process.applicablePid"
+      ],
+      "parent_ref.name": [
+        "Process.parentProcess"
+      ],
+      "binary_ref.name": [
+        "Process.elementDisplayName"
+      ]
+    }
+  },
+  "user-account": {
+    "fields": {
+      "user_id": [
+        "Machine.adSid",
+        "Process.user",
+        "Process.hostUser",
+        "User.elementDisplayName"
+      ],
+      "account_login": [
+        "User.adLogonName",
+        "LogonSession.user",
+        "LogonSession.winLogonDetails"
+      ],
+      "is_privileged": [
+        "User.isAdmin"
+      ],
+      "display_name": [
+        "User.elementDisplayName",
+        "User.adDisplayName",
+        "User.adCanonicalName"
+      ]
+    }
+  },
+  "file": {
+    "fields": {
+      "name": [
+        "File.elementDisplayName",
+        "File.originalFileName",
+        "DetectionEvents.file",
+        "Driver.file",
+        "Service.binaryFile",
+        "Service.oldBinaryFile"
+      ],
+      "size": [
+        "File.size"
+      ],
+      "hashes.MD5": [
+        "File.md5String"
+
+      ],
+      "hashes.'SHA-1'": [
+        "File.sha1String"
+
+      ],
+      "hashes.'SHA-256'": [
+        "File.sha256String"
+      ],
+      "parent_directory_ref.path": [
+        "File.correctedPath",
+        "File.path",
+        "FileAccessEvent.path",
+        "Service.unitFilePath"
+      ]
+    }
+  },
+  "directory": {
+    "fields": {
+      "path": [
+        "File.correctedPath",
+        "File.path",
+        "FileAccessEvent.path",
+        "Service.unitFilePath"
+      ]
+    }
+  },
+  "windows-registry-key": {
+    "fields": {
+      "key": [
+        "Autorun.elementDisplayName"
+      ],
+      "values[*]": [
+        "Autorun.value"
+      ]
+    }
+  },
+  "x-cybereason-file": {
```

Comment:
Why is all of this being assigned to a custom object? Most of these should be going in the file object and anything properties that aren't standard can go in as extensions to the STIX object.

Reply:
Removed all the repeated attributes from the customs object in from_stix_map.json